view/administration/owners/editOwner/editOwner.php

```diff
@@ -50,24 +50,139 @@
                     </div>
                 </div>
                 <div class="col-md-3">
                     <div class="form-group">
                     <!-- DOB section -->
                     <label for="email">Email:</label>
-                        <input type="text" class="form-control" id="email">
+                        <input v-model="owner.email" type="text" class="form-control" id="email">
                     </div>
                 </div>
             </div> 
             <div class="row">
                 <div class="col-md-3">
-                    <button type="button" class="btn btn-success bottom-form-button">Edit</button>
+                    <button @click="editOwner" type="button" class="btn btn-success bottom-form-button">Edit</button>
                 </div>
             </div>
         </div>
     </div>
+        <!-- Error Messages -->
+    <div v-for="(error, index) in errors" 
+         :key="index" class="alert alert-danger alert-dismissible fade show message-box" 
+         >
+        <button type="button" class="close" @click="clearError(index)">&times;</button>
+        {{error}}
+    </div>
+
+    <!-- Success Message -->
+    <div v-if="successMessage != null" 
+         class="alert alert-success alert-dismissible fade show message-box" 
+         >
+        <button type="button" class="close" @click="clearSuccessMessage">&times;</button>
+        {{successMessage}}
+    </div>
 </div>
 
-<link rel="stylesheet" href="../view/administration/owners/editOwner/editOwner.css">
\ No newline at end of file
+<link rel="stylesheet" href="../view/administration/owners/editOwner/editOwner.css">
+<style scoped>
+.message-box {
+    position: fixed;
+    bottom: 0;
+    right: 5px;
+    width: 300px;
+}
+</style>
+<script>
+    /* script for binding data */
+    new Vue({
+            el: "#editOwnerApp",
+            data: {
+                id: null,
+                owner:{
+                    firstName: null,
+                    middleName: null,
+                    lastName: null,
+                    phoneNumber: null,
+                    address: null,
+                    email: null
+                },
+                errors: [],
+                successMessage: null
+            },
+            created() {
+                this.getId();
+                this.getOwner();
+            },
+            methods: {
+                getId(){
+                let uri = window.location.href.split('?');
+                if(uri.length === 2) {
+                  let vars = uri[1].split('&');
+                  let getVars = {};
+                  let tmp = '';
+                  vars.forEach(function(v) {
+                    tmp = v.split('=');
+                    if(tmp.length === 2)
+                      getVars[tmp[0]] = tmp[1];
+                  });
+                  this.id = getVars['id'];
+                }
+                },
```

Comment:
Fix indentation. My suggestion is to select lines 132 - 183 and do a `SHIFT` `TAB`